app/controllers/admins/confirmations_controller.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
```

Comment:
I’m confused by a of these controllers.  They all look like they are intended to do the same thing.  Confirmations, OmniauthCallbacks, Registrations, and Sessions.   Did you somehow generate the controllers for Devise?  If so, we for sure don’t want to do that.